db/mainphotos.cql

```diff
@@ -33,10 +33,10 @@ CREATE TABLE mainphotos.price_history (
 
 DROP TABLE IF EXISTS mainphotos.property_images;
 CREATE TABLE mainphotos.property_images (
-  image_id int,
+  image_id uuid,
```

Comment:
are you adding uuid to update records easily?

Reply:
Initially the id was going to be an autoincrementing value but since cassandra doesn't work well with those I decided on using a uuid as a primary key identifier